emm_private.c

```diff
@@ -72,13 +72,13 @@ int mm_init_ema(void *addr, size_t size, int flags, int prot,
 
 extern int mm_alloc_internal(void *addr, size_t size, uint32_t flags,
                  sgx_enclave_fault_handler_t handler,
-                 void *private, void** out_addr, ema_root_t* root);
+                 void *priv, void** out_addr, ema_root_t* root);
 
 int mm_alloc(void *addr, size_t size, uint32_t flags,
                  sgx_enclave_fault_handler_t handler,
-                 void *private, void** out_addr)
+                 void *priv, void** out_addr)
 {
-    return mm_alloc_internal(addr, size, flags, handler, private,
+    return mm_alloc_internal(addr, size, flags, handler, priv,
                             out_addr, &g_rts_ema_root);
```

Comment:
we need check if requested range is in [user_start, user_end). And return EINVAL if so. see spec for sgx_mm_init
Need do same check for mm_init_ema function above